### PR TITLE
Fixes #17: Delete existing Experience 

### LINKED ISSUE #5: Implement API route to return all "Experiences"

### LINKED ISSUE #17: Delete existing Experience

### PR DESCRIPTION
**Pull request by haianhng31**
resolves #17

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -49,6 +49,10 @@ def experience():
 
     if request.method == 'POST':
         return jsonify({})
+    
+    if request.method == 'DELETE':
+        data["experience"] = []
+        return jsonify(), 204
```
Comment (Mackay-Fisher) [inline app.py:55]: I'm not entirely sure, but when we have a successful deletion are we supposed to return a 200 status code? Then if there is no information return 204? I just finished implementing them myself for the other sections and did it that way. Please let me know my background is not web dev lol.

File: app.py [hunk 2]
```diff
@@ -49,6 +49,10 @@ def experience():
 
     if request.method == 'POST':
         return jsonify({})
+    
+    if request.method == 'DELETE':
+        data["experience"] = []
```
Comment (Mackay-Fisher) [inline app.py:54]: Hey, I think we are supposed to delete based on an index not just for all of the experiences:
https://github.com/MLH-Fellowship/orientation-project-python-24.FAL.B/issues/5

Comment (haianhng31) [reply app.py]: Thks for pointing out. I have updated the code!

**Comment by haianhng31** [comment]
@Mackay-Fisher @16BitNarwhal can u help me review and merge this so i can continue fix merge conflicts in my other PRs?

**Comment by Mackay-Fisher** [comment]
LGTM!

**Review by 16BitNarwhal — APPROVED** [review]

**Pull request merged**